index.js

```diff
@@ -91,7 +91,7 @@ app.use((err, req, res) => {
 });
 
 // finally, let's start our server...
-const server = app.listen(process.env.PORT || 3000, () => {
+const server = app.listen(process.env.PORT || 7000, () => {
```

Comment:
Any reason for changing the port number from 3000 to 7000? If none please leave it at 3000

Reply:
Thank you @Oloyedesinmiloluwa . I have made the required changes.